docs/terms_and_conditions.md

```diff
@@ -0,0 +1,148 @@
+e. szkody wynikłe na skutek braku ciągłości dostarczania Usług, będące następstwem okoliczności, za które Usługodawca nie ponosi odpowiedzialności (siła wyższa, działania i zaniechania osób trzecich itp.),
+f. podania przez Użytkownika i Użytkownika niezalogowanego nieprawdziwych, nieaktualnych lub niepełnych danych lub informacji,
+g. nieprzestrzeganie przez Użytkownika i Użytkownika niezalogowanego postanowień Regulaminu.
+
+**VIII. OCHRONA DANYCH OSOBOWYCH** (w Polityce Prywatności???)
+1. Każdy Użytkownik oraz Użytkownik niezalogowany ma prawo do ochrony jego prywatności przez Usługodawcę.
+2. Zasady dotyczące przetwarzania przez Usługodawcę danych osobowych Użytkowników oraz Użytkowników niezalogowanych określa Polityka Prywatności.
+
+**IX. POSTĘPOWANIE REKLAMACYJNE** (???)
+1. Użytkownik i Użytkownik niezalogowany mają prawo składać reklamacje w sprawach dotyczących realizacji Usług.
+2. Reklamacje rozpatruje Usługodawca.
+3. Prawidłowo złożona reklamacja powinna zawierać co najmniej następujące dane:
+a. oznaczenie Użytkownika lub Użytkownika niezalogowanego (imię, nazwisko, adres poczty elektronicznej),
+b. adres Serwisu, którego reklamacja dotyczy,
+c. przedmiot reklamacji,
+d. okoliczności uzasadniające reklamację.
+4. Reklamacje nie zawierające powyższych danych nie będą rozpatrywane przez Usługodawcę.
+5. Reklamację (z dopiskiem „Reklamacja”) należy składać:
+a. na adres Usługodawcy: Fundacja Code for Poznań., ul. Takaitaka 99/100, 00-000 Poznań,
+b. za pośrednictwem dostępnego w Serwisach formularza, lub
+c. na adres poczty elektronicznej: pomoc@volontulo.pl.
+6. Usługodawca dołoży starań, aby reklamacje były rozpatrzone w terminie najpóźniej 14 dni od ich otrzymania od Użytkownika lub Użytkownika niezalogowanego. O swojej decyzji zapadłej w wyniku rozpatrzenia reklamacji Usługodawca zawiadomi Użytkownika lub Użytkownika niezalogowanego za pośrednictwem poczty elektronicznej, na adres podany w reklamacji zgodnie z ust. 3 powyżej.
+7. Reklamacja rozpatrzona zgodnie z postanowieniami Regulaminu nie podlega dalszemu ani ponownemu rozpatrzeniu.
+
+**X. KONTAKT Z USŁUGODAWCĄ**
+Użytkownik oraz Użytkownik niezalogowany mogą kontaktować się z Usługodawcą we wszelkich sprawach związanych z korzystaniem z Serwisu: pisemnie na adres Usługodawcy: Fundacja Code for Poznań., ul. Takaitaka 99/100, 00-000 Poznań lub mailowo na adres: pomoc@volontulo.pl.
+
+**XI. ROZWIĄZANIE UMOWY LUB ODSTĄPIENIE OD UMOWY**
+1. Umowa o świadczenie Usług może być rozwiązana przez którąkolwiek ze stron.
+2. W dowolnym momencie Użytkownik ma prawo rozwiązać umowę o świadczenie Usług poprzez samodzielne usunięcie swojego Konta, a Użytkownik niezalogowany poprzez zaprzestanie korzystania z Usług.
```

Comment:
Aktualnie chyba nie ma możliwości samodzielnego skasowania konta - trzeba poprosić o to administratora przez formularz kontaktu